azure-pipelines.yml

```diff
@@ -16,8 +16,19 @@ variables:
     value: true
   - name: _BuildConfig
     value: 'Debug'
+  # CodeQL3000 tasks are auto-injected into official builds of main. Variables below are ignored otherwise.
```

Comment:
No, the tasks will be auto-injected into _all_ builds of main. Make `Codeql.Enabled` undefined or `false` in public builds. We don't want or need duplicate bugs.